mappings/net/minecraft/client/gui/screen/world/EditGameRulesScreen.mapping

```diff
@@ -19,70 +19,71 @@ CLASS net/minecraft/unmapped/C_ajacvfls net/minecraft/client/gui/screen/world/Ed
 		ARG 1 button
 	METHOD m_ocffiabp markValid (Lnet/minecraft/unmapped/C_ajacvfls$C_vdrulthz;)V
 		ARG 1 ruleWidget
-	CLASS C_fntallks RuleWidgetFactory
+	CLASS C_fntallks GameRuleEntryFactory
 		METHOD create create (Lnet/minecraft/unmapped/C_rdaqiwdt;Ljava/util/List;Ljava/lang/String;Lnet/minecraft/unmapped/C_xmldumst$C_ubszziqk;)Lnet/minecraft/unmapped/C_ajacvfls$C_vdrulthz;
-			ARG 1 name
-			ARG 2 description
-			ARG 3 ruleName
-			ARG 4 rule
-	CLASS C_lppnmpti NamedRuleWidget
-		FIELD f_ggwwaclx name Ljava/util/List;
+			ARG 1 message
+			ARG 2 tooltip
+			ARG 3 description
+			ARG 4 gameRule
+	CLASS C_lppnmpti GameRuleEntry
+		FIELD f_ggwwaclx message Ljava/util/List;
 		FIELD f_nxcbmpxi children Ljava/util/List;
 		METHOD <init> (Lnet/minecraft/unmapped/C_ajacvfls;Ljava/util/List;Lnet/minecraft/unmapped/C_rdaqiwdt;)V
-			ARG 2 description
-			ARG 3 name
-		METHOD m_getgtnya drawName (Lnet/minecraft/unmapped/C_sedilmty;II)V
-			ARG 2 x
-			ARG 3 y
-	CLASS C_nulgdlho RuleCategoryWidget
-		FIELD f_oqcsioxt name Lnet/minecraft/unmapped/C_rdaqiwdt;
+			ARG 2 tooltip
+			ARG 3 message
+		METHOD m_getgtnya drawMessage (Lnet/minecraft/unmapped/C_sedilmty;II)V
+			ARG 1 graphics
+			ARG 2 y
+			ARG 3 x
```

Comment:
Note that these have been swapped as well.  The method's parameters are indeed `GuiGraphics graphics, int y, int x`.  I assume it's a mistake.  But they are used within the method, which is how you can tell:
```java
protected void drawMessage(GuiGraphics graphics, int y, int x) {
    if (this.name.size() == 1) {
        graphics.drawText(EditGameRulesScreen.this.client.textRenderer, (OrderedText)this.name.get(0), x, y + 5, 16777215, false);
    } else if (this.name.size() >= 2) {
        graphics.drawText(EditGameRulesScreen.this.client.textRenderer, (OrderedText)this.name.get(0), x, y, 16777215, false);
        graphics.drawText(EditGameRulesScreen.this.client.textRenderer, (OrderedText)this.name.get(1), x, y + 10, 16777215, false);
    }
}
```
See how the `x` and `y` are used in the `drawText` calls. The `y` is shifted based on whether there is one or two lines of text.